src/pedalmarking.ts

```diff
@@ -116,11 +123,7 @@ export class PedalMarking extends Element {
     this.custom_depress_text = '';
     this.custom_release_text = '';
 
-    this.font = {
-      family: 'Times New Roman',
-      size: 12,
-      weight: 'italic bold',
```

Comment:
We previously allowed weight to include both "italic" and "bold".... The new Font API mirrors the CSS standard.